### ISSUE TITLE
info about golaxy training

### ISSUE THREAD

**Issue by l1t1** [leela-zero/leela-zero#2400]
http://m.newsmth.net/article/Weiqi/629563

**Comment by iopq** [comment]
我不明白了

**Comment by Vargooo** [comment]
這是什麼意思

**Comment by sbbdms** [comment]
Trying to translate... I hope it is good enough _(:з」∠)_

Title: Visually feels the training progress of Go AI

Golaxy started the training of a model with new architecture on 20th, May. The model has been initialized. Its training progress is observed via regular matches against a baseline model. The baseline model is what Golaxy used in the Fuzhou AI competition at the end of April.

The 1st picture is a game played by the model after less than an hour.

The 2nd picture is a game played by the model after one day, it has a 1.9% winrate against the baseline model.

The 3rd picture is a game played by the model on the 3rd day, it has a 5.0% winrate against the baseline model. Its strength is considered to surpass all human players.

The 4th picture is a game played by the model on the 4th day, the winrate is 7.1%.

The 5th picture is a game played by the most recent model, the winrate is 9.6%.

The training above is SL (using existed games), its speed is considered to be usual.
When it has approximate strength of the baseline model, the training will be shifted to RL.

8 GPUs are used in training, each GPU has approximate FLOPS of P40.

![snap1](https://user-images.githubusercontent.com/19167292/58311002-8ff65800-7e3a-11e9-832a-55cdeec8dd84.jpg)
![snap2](https://user-images.githubusercontent.com/19167292/58311004-908eee80-7e3a-11e9-96a2-3a38f39c258e.jpg)
![snap3](https://user-images.githubusercontent.com/19167292/58311006-908eee80-7e3a-11e9-9256-ce876b46a89b.jpg)
![snap4](https://user-images.githubusercontent.com/19167292/58311007-908eee80-7e3a-11e9-8f9c-9dfa5d8c5e87.jpg)
![snap5](https://user-images.githubusercontent.com/19167292/58311010-91278500-7e3a-11e9-97a6-2e759fec77d6.jpg)

**Comment by Vargooo** [comment]
Thx!

**Comment by Splee99** [comment]
So in summary it started from scratch using supervised learning with the existing games of previous Golaxy ?

**Comment by kfc51151271** [comment]
Golaxy started a new training with a new network type(probably the SE network) with 10 million training data.
Golaxy is a well-known commercial Go project in China that frequently appears in various computer Go tournaments

**Comment by luckyangman** [comment]
Golaxy won't make amazing progress. Just copy the test!

**Comment by bash99** [comment]
Some additional Info:
The 4th picture is the Mi's fly knife (芈氏飞刀), and is correct move for it, so it is already very strong in 4 days.

More comments from  GunL (Golaxy team member)：
AZ的方法比AGZ更好。
AZ's way is better than AGZ; (AZ's way mean let any new weight generate self-play game, not just those more strong ones.)

【 在 oraclewq (oraclewq) 的大作中提到: 】
: leela训练一个模型貌似用27万谱。
那大概率过拟合啊。

[ response to oraclewq's saying ]
 "leala train a new weight use about 270K self-play games".
It will very likes be overfitting in training.

数据量相对模型参数数量的比值比较小呀。
[It's because] the ratio of the amount of data to the number of model parameters is relatively small.

不清楚LZ有没有设置测试集，有mse之类的指标对照会比较直观。
Elf v2的论文比较了AGZ和AZ，虽然棋盘路数不同，结论还是靠谱的：AZ优于AGZ。
It is not clear whether LZ has set up a benchmark. It is more intuitive to have a comparison of indicators such as mse.
Elf v2's paper compares AGZ and AZ. Although the size of board is different, the conclusion is reliable: AZ is better than AGZ.

那么至少AZ的方法是可以拿来对照的：训练数据窗口50万谱，来自至少20个不同模型。考虑到AZ的数据窗口滑动非常快，而数据更新速度越快，越能支撑以更小的数据窗口来训练并避免过拟合；采用比AZ更激进的策略就得注意过拟合的问题。
Then at least the AZ method can be compared: the training data window is 500,000 spectra, from at least 20 different models. Considering that AZ's data window slides very fast, and the faster the data is updated, the more it can support training with smaller data windows and avoid over-fitting; using a more aggressive strategy than AZ has to pay attention to the problem of overfitting.

在没有测试集的情况下，过拟合可以从侧面情况反映出来：越来越固化的前半盘；对生成训练数据的模型胜率较高而对较早的模型或第三方基线模型胜率未见提高。
In the absence of a test set, overfitting can be reflected from the side case: the first half of the disk that is getting solidified; the model winning rate for generating training data is higher and the winning rate for earlier models or third-party baseline models is not improved. .
